README.md

```diff
@@ -62,25 +84,25 @@ There are four databases (one for global and three for industries) in this Appli
 | aerospace     | Used to store the data about aerospace industry.     |
 
 ### Connection configuration
-This application exposes port 9001 for the user to connect to the HSQLDB database remotely.
+This application exposes port 9001(customizable) for the user to connect to the HSQLDB database remotely.
 
 - Global database
 
-| Option   | Value                                      |
-|----------|--------------------------------------------|
-| Driver   | `org.hsqldb.jdbcDriver`                    |
-| URL      | `jdbc:hsqldb:hsql://localhost:9001/global` |
-| Username | `SA`                                       |
-| Password | `pass`                                     |
+| Option   | Value                                      | Customizable |
```

Comment:
Change all the "Customizable" column name to be "Property" and change the value to be corresponding property name.